### DISCUSSION TITLE
Add data module to experiment prepared in html/css/vanilla javascript

### DISCUSSION THREAD

**Discussion by dolenda** [jspsych/jsPsych#766] [Q&A] [answered]
Hello,
I prepared a part of a decision task using html/css/vanilla javascript. Then I tried to use Firebase, backend as service, but I'm a bit confused and not sure how to get data from participants. Is it possible to just apply jsPsych.data module to my web app?

Thanks in advance!
Dominik

**Comment by vijaymarupudi** [comment]
I'm not familiar with Firebase, but you can post the data to your database using this perhaps?

https://firebase.google.com/docs/reference/rest/database#section-post

**Reply by dolenda** [reply]
Yes, I'll try with that one.

**Comment by jodeleeuw** [comment, marked as answer]
I think you are asking whether you can use `jsPsych.data` without using the rest of jsPsych. That won't help solve the issue you are having. `jsPsych.data` is an interface to the client-side data generated by jsPsych experiments. It sounds like you are working on a back end service, and jsPsych doesn't provide any standard tools for the back end.

**Reply by dolenda** [reply]
Exactly, this answers my question, thanks! Ok, I'll try with Firebase backend.
